mantidproject/mantid
# Splitters documentation

Migrating more wiki pages to sphinx. This is (mostly) just a reformatting of the original documentation.
- http://www.mantidproject.org/TimeSeriesProperty
- http://www.mantidproject.org/EventFiltering
- http://www.mantidproject.org/SplittersWorkspace
- http://www.mantidproject.org/SplittingInterval (merged into SplittersWorkspace)

**To test:**

Build the documentation and see that it is reasonable.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
